src/mixer/audio.ts

```diff
@@ -323,25 +340,64 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
     this.micMixGain.gain.value = value;
   }
 
-  getLevel(source: LevelsSource) {
+  getLevels(source: LevelsSource, stereo: boolean) {
```

Comment:
Consider explicitly declaring the return type as `[number] | [number, number]`, right now I think it infers it as `number[]`

Reply:
It didn't like that at the other side, so i've just made it [number,number] always.